README.md

```diff
@@ -2676,10 +2676,23 @@ A successful response uses the HTTP code **200 OK** and has a JSON body with the
 | Field                   | Value                           | Description      |
 |:------------------------|:--------------------------------|:-----------------|
 | `result`                | String                          | The value `success` indicates that the body represents a successful response. |
-| `last_datetime`         | Integer                         | The last reported validation vote signed by this validator. |
 | `validation_public_key` | String - Base-58 [Public Key][] | This validator's validator public key. |
 | `domain`                | String                          | (May be omitted) The DNS domain associated with this validator. |
-| `domain_state`          | String                          | The value `verified` indicates that this validator has a [verified domain](https://developers.ripple.com/run-rippled-as-a-validator.html#domain-verification) controlled by the same operator as the validator. The value `AccountDomainNotFound` indicates that the "Account Domain" part of Domain Verification is not set up properly. The value `RippleTxtNotFound` indicates that the ripple.txt step of Domain Verification is not set up properly. |
+| `chain`                 | String                          | Ledger hash chain which this validator is currently following. |
```

Comment:
We should explicitly publish the possible values. The example shows `main`, but what else? Presumably there's one for the test net (or possibly several in preparation for multiple test nets?)

Reply:
the only named ones will be `main` and `altnet` right now - any other will be `chain.[Integer]` where integer is just a sequential number updated whenever any "new" chain is discovered.  This happens a lot with validators that are not well synced